tools/qmctl

```diff
@@ -0,0 +1,352 @@
+            if not stripped or stripped.startswith("#"):
+                continue
+            if stripped.startswith("[") and stripped.endswith("]"):
+                current_section = stripped[1:-1]
+                parsed[current_section] = {}
+            elif "=" in stripped and current_section:
+                key, value = map(str.strip, stripped.split("=", 1))
+                parsed[current_section][key] = value
+            elif current_section:
+                parsed[current_section][stripped] = True
+
+        return dict(parsed)
+
+    def show(self, output_json=False, pretty=True):
+        if not os.path.exists(self.config_path):
+            self._print_output(
+                {"error": f"Configuration file {self.config_path} not found."},
+                output_json,
+                pretty
+            )
+            sys.exit(1)
+
+        self._log_path("Reading", self.config_path)
+        with open(self.config_path, "r") as file:
+            content = file.read()
+
+        if output_json:
+            parsed = self.parse_to_dict(content)
+            self._print_output({"path": self.config_path, "sections": parsed}, output_json, pretty)
+        else:
+            print(content, end="")
+
+    def exec_in_container(self, command, output_json=False, pretty=True):
+        if not self._container_exists(self.container):
+            self._print_output(
+                {"error": f"Container '{self.container}' does not exist."},
+                output_json,
+                pretty
+            )
+            sys.exit(1)
+
+        if not command:
+            self._print_output({"error": "No command provided to execute."}, output_json, pretty)
+            sys.exit(1)
+
+        try:
+            result = subprocess.run(
+                ["podman", "exec", self.container] + command,
+                capture_output=True,
+                text=True,
+                check=False
+            )
+
+            if "command not found" in result.stderr.lower():
+                raise RuntimeError(f"Command '{' '.join(command)}' not found inside the container.")
+
+            self._print_output(
+                {"output": result.stdout.strip()},
+                output_json,
+                pretty
+            )
+
+        except Exception as e:
+            self._print_output({"error": str(e)}, output_json, pretty)
+            sys.exit(1)
+
+    def show_unix_sockets(self, output_json=False, pretty=True):
+        try:
+            result = subprocess.run(
+                ["podman", "exec", self.container, "ss", "-xl"],
+                capture_output=True,
+                text=True,
+                check=False
+            )
+
+            if "command not found" in result.stderr.lower():
+                raise RuntimeError(
+                    "'ss' command not found inside the QM container.\n"
+                    "To install it, run:\n"
+                    "  sudo dnf --installroot=/usr/lib/qm/rootfs/ install ss -y"
```

Comment:
This one is failing 
for fedora cs distros, the package is iproute
dnf --installroot=/usr/lib/qm/rootfs/ install iproute -y